master/internal/stream/system.go

```diff
@@ -178,18 +182,19 @@ func writeAll(socket *websocket.Conn, msgs []*websocket.PreparedMessage) error {
 }
 
 // Websocket is an Echo websocket endpoint.
-func (ps *PublisherSet) Websocket(
+func (ps *PublisherSet) entrypoint(
```

Comment:
Could you also make sure the `Supervisor.Websocket()` is adjusted to reflect this change?

Reply:
Not worries, I was just referring to the rename; I believe the supervisor.Websocket() is calling the ps.Websocket(), which might not exist anymore

Reply:
you mean the publisherSet websocket? That's on line 332